### PR TITLE
Add Russian translation

### PR DESCRIPTION
**Pull request by sshalyminov**
Add Russian translation

### CODE REVIEW

File: src/assets/localization/languages/ru.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "Azimuth": "Азимут",
+  "Dawn": "Рассвет",
+  "Dusk": "Сумерки",
+  "Elevation": "Высота",
+  "Noon": "Полдень",
```
Comment (bessarabov) [inline src/assets/localization/languages/ru.json:6]: What do you think? Maybe it is better to use "Солнечный полдень" for "Noon"?

Screen Shot 2021-05-18 at 16 14 28

https://ru.wikipedia.org/wiki/%D0%9F%D0%BE%D0%BB%D0%B4%D0%B5%D0%BD%D1%8C#%D0%A1%D0%BE%D0%BB%D0%BD%D0%B5%D1%87%D0%BD%D1%8B%D0%B9_%D0%BF%D0%BE%D0%BB%D0%B4%D0%B5%D0%BD%D1%8C

Comment (sshalyminov) [reply src/assets/localization/languages/ru.json:6]: I've never heard anyone talking "Солнечный полдень" - just usual "Полдень" for both 12pm time and for highest Sun position. This is like "масло масляное". But if you wish...

Comment (bessarabov) [reply src/assets/localization/languages/ru.json]: Well, I'm not sure what is the best possible word. But I would prefer not to use word "Полдень" — I think it is confusing to see the time that is not 12:00 near that world.

**Comment by bessarabov** [comment]
Thank you! 👍

File: src/assets/localization/languages/ru.json (hunk 1 above)
Comment (ukko) [reply src/assets/localization/languages/ru.json]: Можно ещё и "Зенит" использовать. Но "Полдень" - самое точное описание.

Там-там-там-тааамм.. В Москве полдень!

**Review by sshalyminov — COMMENTED** [review]
Changed to "Зенит" - more exact term

**Review by AitorDB — APPROVED** [review]

**Pull request merged**